.pre-commit-config.yaml

```diff
@@ -0,0 +1 @@
+/nix/store/yz1wdysfasppp6v1krg8fw9q1pz9hcqi-pre-commit-config.json
```

Comment:
I think this file should be added to .gitignore (it should definitely not be present in git).